reco_utils/evaluation/python_evaluation.py

```diff
@@ -224,7 +237,7 @@ def _merge_ranking_true_pred(
         pd.DataFrame: new data frame of true data DataFrame of recommendation hits
             number of common users
     """
-
+    # Check existence of input columns.
```

Comment:
I don't think we need this checking anymore. Type checking codes you are adding will raise an error if the columns are not in the df.